setup.py

```diff
@@ -51,21 +61,15 @@
     "datasets",
 ]
 
-# CUDA kernels
-if platform.system().lower() != "darwin" and HAS_CUDA:
+# kernels can be downloaded from pypi for cuda+(linux or windows)
+if platform.system().lower() != "darwin" and CUDA_VERSION:
     requirements.append("autoawq-kernels")

```

Comment:
So to clarify, ROCm users will need to install from a github release? Because the kernels distributed on pypi are CUDA only.

Reply:
I think this should be `platform.system().lower() != "darwin" and (CUDA_VERSION or ROCM_VERSION)` since we need the user to still install `autoawq-kernels` for inference.